SoftLayer/CLI/account/orders.py

```diff
@@ -0,0 +1,38 @@
+"""Order list account"""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer.managers.account import AccountManager as AccountManager
+from SoftLayer import utils
+
+
+@click.command()
+@click.option('--limit', '-l',
+              help='How many results to get in one api call, default is 100',
```

Comment:
Dont need `default is 100` in this help text, as the `show_default=True` adds that.

Reply:
set the help message to `help='How many results to get in one api call'`